content/working_with/console/widgets/blueprints.md

```diff
@@ -56,43 +56,45 @@ The default visibility is "Tenant", and according to the logged-in user's permis
 ##### Using Terraform module
 1. Click the **Upload** button.
 2. In the menu, click **Upload from Terraform module** option.
-3. **Create blueprint from Terraform** modal will appear
+3. **Create blueprint from Terraform** modal will appear.
 4. Enter the `Blueprint name`   
-   `Blueprint name` is the name with which you want to identify this blueprint once uploaded
-5. Select `Terraform version` (as a default, one of the terraform versions will be selected)
+   `Blueprint name` is the name with which you want to identify this blueprint once uploaded.
+5. Select `Terraform version` (by default, one of the Terraform versions will be selected).
 6. Provide `Blueprint main information`, which consists of:
-   - `URL to a zip archive that contains the Terraform module`,
+   - `URL to a zip archive that contains the Terraform module`.
    - `Terraform folder in the archive` - it's a selectable field from which you'll be able to select directories found inside the linked zip file.<br />
-   By default this field is disabled, until the `URL to a zip archive that contains the Terraform module` is being provided.
+   By default this field is disabled, until the `URL to a zip archive that contains the Terraform module` is provided.
    - (Optional) Credentials required for accessing `URL to a zip archive that contains the Terraform module`.
-7. (Optional) Add `Variables`, by:   
-   1. Clicking the `Variables` dropdown
-   2. Clicking the **Add** button
+7. (Optional) To add `Variables`:   
+   1. Click the `Variables` select.
+   2. Click the **Add** button.
    3. Filling the row fields:
```

Comment:
```suggestion
   3. Fill the row fields:
```

Reply:
Oh yeah
After changing `clicking` into `click` we introduced this kind of incosistency
I'm correcting it right away!

Reply:
Done 🎉